api/services/Reddit.js

```diff
@@ -79,13 +94,19 @@ exports.getBothFlairs = async function (refreshToken, user) {
   return Promise.all([ptradesFlairPromise, svexFlairPromise]);
 };
 
-exports.setFlair = function (refreshToken, name, cssClass, text, subreddit) {
+exports.setUserFlair = function (refreshToken, name, cssClass, text, subreddit) {
   var actual_sub = sails.config.debug.reddit ? sails.config.debug.subreddit : subreddit;
   var url = 'https://oauth.reddit.com/r/' + actual_sub + '/api/flair';
   var data = {api_type: 'json', css_class: cssClass, name: name, text: text};
   return makeRequest(refreshToken, 'POST', url, data, 5);
 };
 
+exports.setLinkFlair = function (refreshToken, subreddit, link_id, cssClass, text) {
```

Comment:
(I'm not suggesting changing the name of this, because it is how reddit calls it but....)

I really hate how it's called link flair, as we don't have link posts, only self posts. I wish they called it post flair.